### ISSUE TITLE
Missing rviews

### ISSUE THREAD

**Issue by fedhere** [MariumS/PUI2017_mas1300#7]
Marium, read the instructions carefully! I asked for the reviews to be uploaded **in your own repo** - I asked that clearly and repeatedly in the instructions

**Comment by MariumS** [comment]
Yes you are right, it was clear. Even in the email I sent you I'd told you I'd uploaded it to my own repo, I don't know how I didn't. Thank you for letting it slide.

**Comment by MariumS** [comment]
"I wrote the markdown directly on github while in the forked folder, so it is there in my fork, my homework 9 folder, and i submitted a pull request to their original repo." is what I wrote

**Comment by MariumS** [comment]
I was having some trouble pushing and see they are in a local backup of my hw9 folder, but not in the actual. something must have gone wrong at that step. Sorry again
